### ISSUE TITLE
Include Xacro support in the drag and drop example

### ISSUE THREAD

**Issue by gkjohnson** [gkjohnson/urdf-loaders#162]
From here:

https://github.com/gkjohnson/xacro-parser

**Comment by yvanblanchard** [comment]
Hello,
Does it mean that this capability (drag and drop a xacro file) is an on-going feature, and thus not available yet from live demo web page yet ?

**Comment by gkjohnson** [comment]
That's correct - xacro is not supported in the in the drag and drop demo page at the moment. However there are instructions [in the README](https://github.com/gkjohnson/urdf-loaders/tree/master/javascript#from-xacro) on how to use the xacro parser with the urdf loader in your own project, though.
